coa-helm/local_values.yaml

```diff
@@ -0,0 +1,28 @@
+# This is a template.
+# Don't change this file; copy it instead, and fill in the values needed for deployment.
+# Obtain these values from someone on the development team.
```

Comment:
A pattern I've seen used for this is to rename this file to something like `local_values.template.yaml` and add `local_values.yaml` to `.gitignore`. Then people can copy the template and have a local working copy that they don't have to worry about accidentally checking in.

Reply:
it's funny, I considered doing exactly this... I don't know why I changed my mind. I'll make the change.